sources/OS.h

```diff
@@ -1,7 +1,6 @@
 #ifndef _OS_H_
 #define _OS_H_
 
-#include <stdint.h>

```

Comment:
I would prefer to keep the include (which is also included in process.h), because it is used a few lines later (`void os_sleep(uint32_t time);` for `uint32_t`).

I prefer the include to be explicit instead of implicit, this follows the philosophy of Python, that, to my mind, should be applied in many programming languages, not only in Python.

The philosophy of Python (https://www.python.org/dev/peps/pep-0020/) states that `Explicit is better than implicit.`

Reply:
Thanks for your reply,  i understood , i remove this commit .